x/participation/keeper/msg_participate_test.go

```diff
@@ -73,6 +132,22 @@ func Test_msgServer_Participate(t *testing.T) {
 				Bidder:       tt.msg.Participant,
 				MaxBidAmount: sdk.NewIntFromUint64(1000),
```

Comment:
We must change this with the tier `MaxBidAmount`, this is not normal if the tests pass at this point